src/main/java/com/crowdin/event/FileChangeListener.java

```diff
@@ -49,20 +48,28 @@ public void after(List<? extends VFileEvent> events) {
         ProgressManager.getInstance().run(new Task.Backgroundable(this.project, "Crowdin") {
             @Override
             public void run(@NotNull ProgressIndicator indicator) {
-                String sources = PropertyUtil.getPropertyValue(PROPERTY_SOURCES, project);
-                List<String> sourcesList = FileUtil.getSourcesList(sources);
-                String branch = GitUtil.getCurrentBranch(project);
-                Crowdin crowdin = new Crowdin(project);
+                CrowdinProperties properties;
+                try {
+                    properties = CrowdinPropertiesLoader.load(project);
+                } catch (Exception e) {
+                    return;
+                }
+                String branch = properties.isDisabledBranches() ? "" : GitUtil.getCurrentBranch(project);
+                Crowdin crowdin = new Crowdin(project, properties.getProjectId(), properties.getApiToken(), properties.getBaseUrl());
+                List<VirtualFile> filesList = properties.getSourcesWithPatterns().keySet()
```

Comment:
Use more understandable variable name. Because now we have `filesList` and `files`. Both have the same type. Hard to distinguish